src/Analyzers/Core/Analyzers/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.cs

```diff
@@ -216,7 +216,7 @@ private bool IsUnusedParameterCandidate(IParameterSymbol parameter)
                     method.IsAbstract ||
                     method.IsVirtual ||
                     method.IsOverride ||
-                    method.PartialImplementationPart != null ||
```

Comment:
Wasn't sure if I should replace `method.PartialImplementationPart != null` by `method.PartialDefinitionPart != null` or keep both conditions, but it turns out the former condition does not appear to be needed to make the tests pass... Let me know if you think I should put it back.

Reply:
NVM, we do need `method.PartialImplementationPart != null` for certain tests to pass...